tables/tab.sql

```diff
@@ -0,0 +1,5 @@
+create database flocx_market;
```

Comment:
You probably want to use [IF NOT EXISTS](https://mariadb.com/kb/en/library/create-database/#if-not-exists) here to avoid errors if you bring up the database container using an existing volume:

```sql
CREATE DATABASE IF NOT EXISTS flocx_market
```